src/test/java/sboot/example/CsvReaderUtilImplTest.java

```diff
@@ -0,0 +1,43 @@
+package sboot.example;
+
+import com.opencsv.CSVReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.junit.jupiter.api.Test;
+import sboot.example.util.CsvReaderUtil;
+import sboot.example.util.CsvReaderUtilImpl;
+
+
+public class CsvReaderUtilImplTest {
+    private static final String CORRECT_PATH = "src/test/resources/correct_path.csv";
+    private static final String INCORRECT_PATH = "src/test/resources/correct/path.csv";
```

Comment:
you can read file from resources using only file name not the whole path(same about not testing resources).